### PR TITLE
Updated documentation of RequestRenderNextUpdate

### PR DESCRIPTION
**Pull request by fredericv-unity3d**
### Purpose of this PR
Added information to a method documentation for better understanding. (See https://fogbugz.unity3d.com/f/cases/1290521).

---
### Testing status
No code was changed, only the documentation.

---
### Comments to reviewers
Doc: please review the modified documentation.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Runtime/Lighting/Reflection/HDAdditionalReflectionData.cs [hunk 1]
```diff
@@ -27,6 +27,8 @@ public static class HDAdditionalReflectionDataExtensions
         /// be rendered the next time it will influence a camera rendering.
         ///
         /// If the probe don't have a <see cref="HDAdditionalReflectionData"/> component, nothing is done.
+        ///
+        /// Note: A camera is influenced by a probe if the camera's frustum intersects the probe's influence volume.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Runtime/Lighting/Reflection/HDAdditionalReflectionData.cs:31]: Note: If any part of a Camera's frustum intersects a Reflection Probe's influence volume, the Reflection Probe influences the Camera.

**Review by JordanL8 — CHANGES_REQUESTED** [review]
Reworded to use active voice. Can you replace the other one with this too? Thanks

**Comment by sebastienlagarde** [comment]
@fredericv-unity3d  any update?

**Comment by sebastienlagarde** [comment]
@fredericv-unity3d can you update this PR? thanks

**Comment by fredericv-unity3d** [comment]
Sorry for the delay, the github notification was lost somehow

**Review by JordanL8 — APPROVED** [review]
Moved the extra information into remarks and reworded some bits. Looks good to me :)

**Pull request merged**